README.txt

```diff
@@ -35,9 +35,9 @@ Offical page of the project is on:
 2. Requirements
 ---------------
 
-    * Latest Java (At least 1.7, 1.8 is supported)
+    * Latest Java (At least 1.8)
       http://www.oracle.com/technetwork/java/
-    * A servlet container like Tomcat (7.x or later)
+    * A servlet container like Tomcat (8.x or later)
```

Comment:
and it will never have it :)
but it will have tomcat 8, so we're good
fwiw , I used tomcat 7 and tomcat 8 from zip and all just worked